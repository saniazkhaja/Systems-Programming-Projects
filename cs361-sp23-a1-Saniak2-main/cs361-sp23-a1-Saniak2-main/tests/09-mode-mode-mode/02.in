561ca800a32dd2b3fe29611b2e479db16d69a9f3967e007b77cce208d5dbe9a3f1a84ac4f1c384a86b16278c6c46dc38f820bcfe77b249eb1125c4e40ee3d681d88ab47ec57d0f788d14eb78c74ba8100dacc65d451fc3caac3d68415744123a2e6c61ccbb37cfe827a7bdd01498ff883a10e98909810cb3198cf33af0a93e5d

