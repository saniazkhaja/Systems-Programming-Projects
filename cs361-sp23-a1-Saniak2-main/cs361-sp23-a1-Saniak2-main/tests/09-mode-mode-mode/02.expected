validation function failed
1465603166
